cmd/end2endtest/helpers.go

```diff
@@ -770,3 +771,24 @@ func (t *e2eElection) registerAnonAccts(voterAccounts []*ethereum.SignKeys) erro
 	}
 	return nil
 }
+
+// logElection prints the election description in the log.
+func logElection(e *vapi.Election) {
+	log.Debugw("election description",
```

Comment:
IMO the `election description` message is misleading as one can expect some actual description of the election.
I suggest to change it for `election data`. Change the comment accordingly too.